README.md

```diff
@@ -132,6 +132,15 @@ To resolve this, rebuild the Docker image for that specific process. For example
 $ docker-compose up --build frontend
 ```
 
+### Running Docker on Linux
+Installation on [Linux](https://askubuntu.com/questions/938700/how-do-i-install-docker-on-ubuntu-16-04-lts).
```

Comment:
Linux != Ubuntu. The official install instructions for Ubuntu, https://docs.docker.com/install/linux/docker-ce/ubuntu/.